monitoring/ping.go

```diff
@@ -0,0 +1,191 @@
+/*
+Copyright 2019 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package monitoring
+
+import (
+	"context"
+
+	"github.com/gravitational/satellite/agent/health"
+	pb "github.com/gravitational/satellite/agent/proto/agentpb"
+	"github.com/gravitational/trace"
+
+	"github.com/codahale/hdrhistogram"
+	serf "github.com/hashicorp/serf/client"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	pingCheckerID               = "ping-checker" // pingCheckerID specifies the check name
```

Comment:
usually, we place comments on the line above the variable, because Go can process them in godoc.

Reply:
Fixed.